api/config/server.js

```diff
@@ -1,7 +1,7 @@
 module.exports = ({ env }) => ({
   admin: {
     auth: {
-      secret: env('ADMIN_JWT_SECRET'),
+      secret: env('ADMIN_JWT_SECRET', '8Gkwzv3B8BOp5V/oVC1TN9N0Xs/27lZ5+scRM4T+6Fr+JuZshJduPzoEmvgHLQMJqrDlNBYYnyrdAHtcE48vBA=='),
```

Comment:
That's really not a good practice to do this. 
Please add a step tu copy a .env.example file and set some vars instead of setting a default for everyone.

Reply:
OKay but by default, when you clone the repo, 
you run `cd api && yarn && yarn run seed && yarn develop` and when you try to register your admin, you have an error.
`Error: secretOrPrivateKey must have a value`

I added a script in the seed part.